_posts/2018-11-30-foreman-community-newsletter---november-2018.md

```diff
@@ -0,0 +1,117 @@
+Foreman.
+
+So, please, *don't* think you have nothing to add to this discussion -
+you do! Get your submissions in by Monday :)
+
+We'll also have a stand at FOSDEM, and a Hackday after CfgMgmtCamp, so book
+your travel for 2nd-6th of Feb and join us for the whole ride!
+
+### New Releases (1.20 and Katello 3.9)
+
+The latest and greatest Foreman is now out (version 1.20), and as always a
+*huge* thanks to the team and the community for the work that goes into
+building, packaging, and testing these releases. Foreman 1.20 brings many new
+goodies, including UI improvements, a new templating engine, an entirely *new*
+system for templating reports, FIPS compliance and more!
+
+You can find all the juicy details in the [release announcement][1_20_0], and
+as always, if you do find issues then please [report them!][issues]
+
+With a new version of Foreman comes a new version of Katello, and this release
+is no exception. Katello 3.9 is now available, bringing content view export and
+import, system purpose, and modularity integration, as well as well over 100
+fixed bugs. Check out the [release announcement][k_3_9] for more details, and
+do report any [issues][k_issues].
+
+As I already said, many thanks to the contributors to these releases - coders,
+RC testers, documentation writers, and bug/feature reporters. Foreman wouldn't
+be what it is without you!
+
+### UI interaction study needs your help
+
+Tereza from our UI team would like to study how people use the newly redesigned
+Audits page in Foreman 1.20. If you've upgraded, and can spare 20 mins, please
+check out
+[this thread](https://community.theforeman.org/t/foreman-1-20-0-and-audits-redesign-user-testing/11861)
+for details on how you can be involved. Thanks!
+
+### [Upcoming Events](https://community.theforeman.org/c/events/l/calendar)
+
+It's pretty quiet this month, with all the holidays, travel, etc, that happens around December...
+
+All Foreman events are available via [the forum
+calendar](https://community.theforeman.org/calendar) - hit the Subscribe button
+(which will give you an ICS link). You can also use the Add To Calendar button
+on specific events if you don't wish to import the whole thing.
+
+Future community demos will now be created much further in advance, so you
+should be able to keep up with upcoming demos there.
+
+* [Community Demo #54](https://community.theforeman.org/t/foreman-community-demo-54)
+
+All the usual goodness from the community, rounded up and presented for your viewing pleasure.
+
+* [FOSDEM 2019](https://community.theforeman.org/t/fosdem-2019/11964)
+
+Our usual yearly meetup, we'll be there with a stand, and plenty of folk to
+talk about Foreman's latest & greatest. Stop by and say hi!
+
+* [CfgMgmtCamp 2019](https://community.theforeman.org/t/config-management-camp/11966)
+
+Foreman will be at CfgMgmtCamp 2019, with booth, talks, demos, swag, and
+probably dinner out on Tuesday night. Join us for our flagship event!
+
```

Comment:
missing hack day info? Also perhaps mentions the event dates again here

Reply:
Thanks @tbrisker updated.